src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFUtility.java

```diff
@@ -7,13 +7,20 @@
 import org.apache.pdfbox.pdmodel.interactive.action.*;
 import org.apache.pdfbox.pdmodel.interactive.annotation.*;
 import org.apache.pdfbox.pdmodel.interactive.documentnavigation.destination.*;
+import org.apache.pdfbox.pdmodel.interactive.documentnavigation.outline.PDOutlineItem;
 import org.springframework.data.util.Pair;
 import uk.gov.hmcts.reform.em.stitching.domain.enumeration.PaginationStyle;
 
 import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
 
 public final class PDFUtility {
-    public static final int LINE_HEIGHT = 15;
```

Comment:
what's wrong with the final here?

Reply:
reverted to same with final, have removed in the initial logic development which was changed later. Thanks